samples/streamlit-in-snowflake/sis-local/README.md

```diff
@@ -33,11 +33,9 @@ from snowflake.snowpark.exceptions import SnowparkSessionException
 
 
 def running_in_sis() -> bool:
-    try:
-        sess = get_active_session()
-        return True
-    except SnowparkSessionException:
-        return False
+    import snowflake.connector.connection
+    import inspect
+    return 1 == len([x for x in inspect.getmembers(snowflake.connector.connection) if x[0] == 'StoredProcConnection'])
```

Comment:
Sweet. I assume this is much faster than trying `get_active_session()`. Would you mind adding a comment to explain this? Why does `snowflake.connector.connection.StoredProcConnection` existing mean it's in SiS?

Reply:
Added a comment and switched it to the inverse (`SnowflakeConnection` exists externally, `StoredProcConnection` exists internally).

Apparently there are (rare) situations where `get_active_session()` will return true outside of SiS as well. So this is faster and more consistent.